src/components/layout/NavigationBar.js

```diff
@@ -16,7 +17,7 @@ export default function NavigationBar() {
   return (
     <Navbar bg="primary" expand="lg">
       <Container>
-        <Navbar.Brand href="/">PawHub</Navbar.Brand>
+        <Navbar.Brand href="/"><Image src={Logo}/></Navbar.Brand>
```

Comment:
We can add a height and width here

`         <Image height={40} width={40} src={Logo} />`